bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/collectionsPanel.js

```diff
@@ -53,7 +53,7 @@ pimcore.object.classificationstore.collectionsPanel = Class.create({
             readerFields.push(columnConfig);
         }
 
-        var url = "/admin/classificationstore/collection-relations?";
+        var url = Routing.generate('pimcore_admin_dataobject_classificationstore_collectionrelations');
```

Comment:
No route found for "GET /admin/classificationstore/collection-relationsxaction=read"

Reply:
done